core/templates/maya.yml

```diff
@@ -23,6 +23,19 @@ keys:
             ass: Arnold (.ass)
         default: ass
 
+    palette_name:
+        type: str
+        filter_by: '^[a-zA-Z](\w+)?[a-zA-Z0-9]$'
+
+    description_name:
+        type: str
+        filter_by: '^[a-zA-Z](\w+)?[a-zA-Z0-9]$'
+
+    patch_name:
+        type: str
+        filter_by: '^[a-zA-Z](\w+)?[a-zA-Z0-9]$'
+
```

Comment:
Are these keys in use? If not, please remove them.

Reply:
Ok. This was added considering the Loader requirements but yeah, removing for now as we can add it later whenever required.